README.md

```diff
@@ -273,6 +273,7 @@ Thanks. please refer to [this](https://github.com/aborazmeh/awesome-palestine/bl
 ## APIs
 
 - 🌐 [List of API endpoints providing data related to Palestine.](https://github.com/Zain-ul-din/awesome-palestine-apis)
+- 🌐 [data.techforpalestine.org](https://data.techforpalestine.org)
```

Comment:
```suggestion
- 🌐 [Palestine Datasets by TFP](https://data.techforpalestine.org)
```

Reply:
just a suggestion. Always good to have a link text that describes the link rather than have the link as the text